modules/meshcloud-idp-lookup-service-principal/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "service_principal_name_suffix" {
+  type        = string
+  description = "Service principal name suffix."
+}
+
+variable "scope" {
+  type        = string
+  description = "The scope to which Service Principal permissions should be assigned to. Usually this is a management group that sits atop the subscriptions."
```

Comment:
scopes always have the form like `/providers/microsoft.management/managementGroups/ContosoCorporate"` right? Might be useful to add that so that consumers don't accidentally put it in e.g. just a UUID